python/simulation.py

```diff
@@ -2107,3 +2107,11 @@ def interpolate(n, nums):
             res.extend(np.linspace(low, high, n + 1, endpoint=False).tolist())
 
     return res + [nums[-1]]
+
+
+def meep_vec(it):
+    try:
+        return mp.vec(*[i for i in it])
```

Comment:
Can't you just do `mp.vec(*it)`?

Reply:
This wasn't working before, but turned out to be a symptom of a different error. Fixed.